react/RatingSummary.tsx

```diff
@@ -136,12 +136,30 @@ const Reviews: FunctionComponent<ReviewProps> = props => {
           })}
         </div>
       </div>
-      <span className="review__rating--average dib v-mid">
-        {average.toFixed(1)}
+      <span className="review__rating--average dib v-mid gray f6">
```

Comment:
Use `c-muted-2` instead of `gray`. 
https://styleguide.vtex.com/#/Styles?id=colors

```suggestion
      <span className="review__rating--average dib v-mid c-muted-2 f6">
```

`gray` is deprecated